src/components/PlantsV2/index.tsx

```diff
@@ -170,7 +170,7 @@ export default function PlantsDashboardV2(): JSX.Element {
         latestObservation?.completedTime
           ? (strings.formatString(
               strings.DASHBOARD_HEADER_TEXT,
-              <b>{strings.formatString(strings.SAMPLE_OF, 2000)}</b>,
+              <b>2000</b>,
```

Comment:
were you able to find out where to get this info from? is that why this is still in draft more?

Reply:
I still don't know. The planting site hectares doesn't make sense here, because the text says "a sample of ... based on last observation", maybe I should sum up all plantingZones' areas of the lastObservation, but still waiting confirmation. And yes, that's why I left it as draft

Reply:
Got it, this could be the area of all monitoring plots that were sampled in this observation round. Maybe sum the number of monitoring plots in the observation results.. and multiply that by the area of a monitoring plot? I think the area is standard (get it from Alex or Steve maybe).

Reply:
Thanks Karthik! Should I wait for a confirmation on how to calculate it or just do it that way?

Reply:
You can mention this approach and confirm it. My understanding is as such.